hack/test-cmd.sh

```diff
@@ -97,6 +97,10 @@ os::test::junit::declare_suite_start "setup/localup"
 os::cmd::try_until_text "oc get --raw /healthz --as system:unauthenticated --config='${ADMIN_KUBECONFIG}'" 'ok' $(( 160 * second )) 0.25
 os::cmd::try_until_success "oc get service kubernetes --namespace default --config='${ADMIN_KUBECONFIG}'" $(( 160 * second )) 0.25
 os::cmd::try_until_success "oc login --server=${KUBERNETES_MASTER} --certificate-authority ${MASTER_CONFIG_DIR}/server-ca.crt -u test-user -p anything" $(( 160 * second )) 0.25
+# install cluster resource quota CRD, otherwise creating project will fail on caches not synced for cluster resource quota
+os::cmd::try_until_success "oc create -f ${OS_ROOT}/test/integration/testdata/crds/clusterresourcequota.yaml --config='${ADMIN_KUBECONFIG}'" $(( 160 * second )) 0.25
```

Comment:
i would expect this in the spot where we create the openshift apiservices.  Probably in our hacked launch